CfgLoadouts.hpp

```diff
@@ -9,20 +9,46 @@ class CfgLoadouts {
   // Allow changeable optics on a global level, note: optic options obey allowMagnifiedOptics rules
   allowChangeableOptics = 1;
 
-  // Do Vehicle Loadouts
-  // (1 will run normaly, 0 will leave them to vanilla defaults, -1 will clear and leave empty)
+  // Vehicle Loadouts -- see "us_mx_mtp.hpp" for examples
+  // -1. Clear and leave inventories empty empty
+  //  0. Leave them to vanilla defaults
+  //  1. Defined supplies are put into vehicle's inventory
+  //  2. Store supplies from vehicle inventory into ammo boxes in vehicle cargo
```

Comment:
Recommend normalizing language: "[...] in vehicle cargo" -> "[...] in vehicle's ACE cargo".

Reply:
I think it's clear by delineating "inventory" vs "cargo", but if you really want it I can change it when I get home.

Reply:
Tbh it only really caught my eye because the other, similar lines all read "vehicle's ACE cargo" and gave me reason to pause

But yeah, just a quick fix and we good imo